samples/python/odometry.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import numpy as np
+import cv2 as cv
+
+import argparse
+import sys
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--algo',
+        help="""DEPTH - works with depth,
+                RGB - works with images,
+                RGB_DEPTH - works with all,
+                defalt - runs all algos""",
+        default="")
+    parser.add_argument(
+        '-src_d',
+        '--source_depth_frame',
+        default="")
+    parser.add_argument(
+        '-dst_d',
+        '--destination_depth_frame',
+        default="")
+    parser.add_argument(
+        '-src_rgb',
+        '--source_rgb_frame',
+        default="")
+    parser.add_argument(
+        '-dst_rgb',
+        '--destination_rgb_frame',
+        default="")
+
+    args = parser.parse_args()
+
+    depth1 = cv.imread(args.source_depth_frame, cv.IMREAD_ANYDEPTH).astype(np.float32)
+    depth2 = cv.imread(args.destination_depth_frame, cv.IMREAD_ANYDEPTH).astype(np.float32)
```

Comment:
These files are not used for "RGB" mode. There is no sense to request them from user.

Similar note is about rgb data.

Reply:
done